1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,153 @@
+            | Project Managers     |
+            | Project Participants |
+        And they see Buttons:
+            | Add Project Manager     |
+            | Add Project Participant |
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Save Button
+        And they see a Next Button
+        Examples:
+            | Role                  |
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+
+    @xxxa
+    Scenario Outline: User opens Dialog for adding Project Manager
+        Given User opens Participants tab for Project
+        And User has the "<Role>" role in the project's scope
+        When they click the Add Project Manager Button
+        Then they see fields:
+            | Start Date  |
+            | User search |
+        Examples:
+            | Role            |
+            | Curator         |
+            | Project Owner   |
+            | Project Manager |
+
+    @xxxb
+    Scenario: User adds a Project Manager
+        Given User opens Dialog for adding Project Manager
+        When they select a Start Date
+        And they enter a name in the User search field
+        And they select a User from the search
+        And they click the Add Button
+        Then they see the User listed as a Project Manager
+
+
+    Scenario Outline: User opens Dialog for adding Project Participant
+        Given User opens Participants tab for Project
+        And User has the "<Role>" role in the project's scope
+        When they click the Add Project Participant Button
+        Then they can select role to be either of:
+            | Project Partcipant    |
+            | Local Project Manager |
+        And they can see User search field
+        Examples:
+            | Role                  |
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+
+    @xxx
+    Scenario: User adds a Project Participant
+        Given User opens Dialog for adding Project Participant
+        When they enter a name in the User search field
+        And they select a role
+        And they select a User from the search
+        And they click the Add Button
+        Then they see the User listed as a Project Participant
+
+    @xxx
+    Scenario Outline: User opens Financing tab for Project
+        Given User sees the Project Wizard
+        And User has the "<Role>" role in the project's scope
+        When they click the Financing tab
+        Then they see fields for:
+            | Financing Code |
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Publish Button
+        Examples:
+            | Role            |
+            | Curator         |
+            | Project Owner   |
+            | Project Manager |
```

Comment:
Er `Local Project Manager` bevisst utelatt? hvordan skal i så fall Financing tab se ut for de?

Reply:
Ja, Local PM skal ikkje gjere noko med finansiering